arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
Fair point. I believe however that in the future we will likely want to support changing processor clock frequency on the fly, so we would rather need to know VDDIO/VDDCORE values. Providing FWS as a config option is not the best approach here. Anyway, that's why it was left as a TODO, we need to figure out how to do it best.